docs/reqs/reqs.tex

```diff
@@ -232,6 +238,25 @@ \section{Non-Functional Requirements}\label{section:nonfuncrec}
 
 TODO: We might have to split non-functional requirements into nf-reqs and constraints (the capstone docs say they're separate, but PC's don't).
 
+% Camera and ML points
+\subsection{Camera}
+\begin{enumerate}[{NF.CAM}.1]
+	\item The cameral must use a standard interface, for future replacability
+	\item The camera must be capable of focusing on objects at variable distances, between 1 and 30 meters.
+	\item The camera must resolve individual objects from a minimum height of 10 meters to a precision of 10cm across.
+	\item The camera must support video capture at a minimum rate of 10 frames per second.
+	\item The camera must support a minimum resolution of 640x480.
+\end{enumerate}
+
+\subsection{Machine Learning Implementation}
+\begin{enumerate}[{NF.ML}.1]
+	\item The machine learning model must be implemented for novel video processing applications.
```

Comment:
Reminder: functional requirements we requirements that describe what a thing does, and a non functional requirement how a thing works. This line sounds more like a functional requirement